projects/plugins/protect/src/js/state/selectors.js

```diff
@@ -1,6 +1,6 @@
 const selectors = {
-	getCredentialState: state => state.credentialState || {},
-	getCredentialStateIsFetching: state => state.credentialStateIsFetching || false,
+	getCredential: state => state.Credential || {},
```

Comment:
```suggestion
	getCredential: state => state.Credential || [],
```
^ Updates the default value to the expected data type.